core/src/main/resources/hudson/slaves/Cloud/sidepanel.jelly

```diff
@@ -33,5 +33,8 @@ THE SOFTWARE.
             <l:task href="delete" icon="icon-edit-delete icon-md" permission="${app.ADMINISTER}" title="${%Delete Cloud}"/>
             <t:actions />
         </l:tasks>
+        <j:forEach var="action" items="${it.allActions}">
+            <st:include it="${action}" page="box.jelly" optional="true"/>
```

Comment:
do you need this for something @Vlatombe? executors just seemed distracting and adding no values on these pages.

Reply:
Used by cloud-stats -- https://github.com/jenkinsci/cloud-stats-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/cloudstats/CloudAction/box.groovy